.lintstagedrc.js

```diff
@@ -56,9 +56,9 @@ module.exports = {
 
     if (files.length > 10) {
       completedStages.add('css');
-      return 'pnpm run stylelint --fix .';
+      return 'pnpm run lint:css';
```

Comment:
```suggestion
      return 'pnpm run stylelint --fix .';
```
Why not fix here?

Reply:
Oliver I think a mistake is in your suggestion and I may not have noticed it in the initial PR for stylelint. The `run` command is used to run scripts listed in `package.json`.

Reply:
Thanks, Anna. You're quite right and I've adjusted the command accordingly.

Reply:
I still think it's worth trying to fix if possible. `stylelint --fix .` should work, too.

Reply:
I'm using the fix command if there's less than ten files. If there's more than 10, I am linting them.

Reply:
Yeah, I see, but why not always fix?

Reply:
Because it does something weird to the package.json

Reply:
```suggestion
      return 'pnpm stylelint --fix ./**/*.css';
```

Then we should be able to tell it to run only on CSS files.

Reply:
![Formatting broken](https://github.com/user-attachments/assets/9ae3d4cf-2e18-4668-9538-bd66d0ec531a)

Reply:
As proven the formatter will touch files that aren't CSS if called directly.

Reply:
Because the `lint:css` command scopes the glob pattern to CSS files. `pnpm stylelint --fix .` does not.

But `pnpm stylelint --fix ./**/*.css` does.

Reply:
That I will allow